recipe/meta.yaml

```diff
@@ -6,63 +6,65 @@ package:
   version: {{ version }}
 
 source:
-  fn: opencv-{{ version }}.tar.gz
-  url: https://github.com/opencv/opencv/archive/{{ version }}.tar.gz
-  sha256: f1b87684d75496a1054405ae3ee0b6573acaf3dad39eaf4f1d66fdd7e03dc852
+  - url: https://github.com/opencv/opencv/archive/{{ version }}.tar.gz
+    fn: opencv-{{ version }}.tar.gz
+    sha256: f1b87684d75496a1054405ae3ee0b6573acaf3dad39eaf4f1d66fdd7e03dc852
+    patches:  # [win]
+      - 01_win_fix_cmake_config_sufix.patch  # [win]
+  - url: https://github.com/opencv/opencv_contrib/archive/{{ version }}.tar.gz
+    fn: opencv_contrib-{{ version }}.tar.gz
+    sha256: 298c69ee006d7675e1ff9d371ba8b0d9e7e88374bb7ba0f9d0789851d352ec6e
+    folder: opencv_contrib-{{ version }}
 
 build:
-  number: 200
+  number: 204
```

Comment:
Should just be 201.

ref: https://conda-forge.org/docs/blas.html

Reply:
Ok, I've read the docs, but don't get it. I really need to understand it. How does it deal with the variants ?
How does it impact at installation time which variant will be chosen, or how will we modify a variant ?
(maybe not the right question for this comment ... )